### PR TITLE
Grammar and consistency in word substitutions

### PR DESCRIPTION
**Pull request by mitchellevan**

### CODE REVIEW

File: comments-by-guideline-and-success-criterion.md [hunk 1]
```diff
@@ -392,7 +392,7 @@ See also the [Comments on Closed Functionality](#comments-on-closed-functionalit
 
 ###### Applying SC 2.1.2 No Keyboard Trap to Non-Web Documents and Software
 
-This applies directly as written, and as described in [Intent from Understanding Success Criterion 2.1.2](https://www.w3.org/WAI/WCAG22/Understanding/no-keyboard-trap#intent), replacing “page” with “non-web document or software”, and “on the Web page” with  "in the non-web document or software", and removing “See Conformance Requirement 5: Non-Interference”.
+This applies directly as written, and as described in [Intent from Understanding Success Criterion 2.1.2](https://www.w3.org/WAI/WCAG22/Understanding/no-keyboard-trap#intent), replacing “page” with “non-web document or software”, replacing “on the Web page” with "in the non-web document or software", and removing “See Conformance Requirement 5: Non-Interference”.
```
Comment (maryjom) [inline comments-by-guideline-and-success-criterion.md:395]:
```suggestion
This applies directly as written, and as described in [Intent from Understanding Success Criterion 2.1.2](https://www.w3.org/WAI/WCAG22/Understanding/no-keyboard-trap#intent) and “page” with “non-web document or software”, replacing “on the Web page” with "in the non-web document or software"; and removing “See Conformance Requirement 5: Non-Interference”.
```

Comment (maryjom) [inline comments-by-guideline-and-success-criterion.md:395]:
```suggestion
This applies directly as written, and as described in [Intent from Understanding Success Criterion 2.1.2](https://www.w3.org/WAI/WCAG22/Understanding/no-keyboard-trap#intent), replacing “page” with “non-web document or software” and “on the Web page” with "in the non-web document or software"; and removing “See Conformance Requirement 5: Non-Interference”.
```

**Comment by mitchellevan** [comment]
FYI, I added the words "and addition" in a couple of spots, like we already had in https://w3c.github.io/wcag2ict/#applying-accessibility-supported-to-non-web-documents-and-software

**Review by bruce-usab — APPROVED** [review]
Thanks Mitch!

**Pull request merged**